test/bdd/steps/network.js

```diff
@@ -441,6 +442,81 @@ Then(/^the last root hash should be the same as one manually calculated$/, async
         .to.be.equal(calculatedDataSetId);
 });
 
+Then(/^the last two exported datasets ([should|should not]+) have the same hashes$/, async function (condition) {
+    this.logger.log('The last root hash should be the same as one manually calculated$');
+    expect(!!this.state.dc, 'DC node not defined. Use other step to define it.').to.be.equal(true);
+    expect(this.state.nodes.length, 'No started nodes').to.be.greaterThan(0);
+    expect(this.state.bootstraps.length, 'No bootstrap nodes').to.be.greaterThan(0);
+    expect(!!this.state.lastExport, 'Last export didn\'t happen. Use other step to do it.').to.be.equal(true);
+    expect(!!this.state.secondLastExport, 'Second last export didn\'t happen. Use other step to do it.').to.be.equal(true);
+    expect(!!this.state.lastImport, 'Last import didn\'t happen. Use other step to do it.').to.be.equal(true);
+    expect(!!this.state.secondLastImport, 'Second last import didn\'t happen. Use other step to do it.').to.be.equal(true);
+
+    const dataset1 = JSON.parse(this.state.secondLastExport.data.formatted_dataset);
+    const dataset2 = JSON.parse(this.state.lastExport.data.formatted_dataset);
+    const dc1 = this.state.nodes[0];
+    let dc2;
+    if (condition.includes('not')) {
+        // eslint-disable-next-line prefer-destructuring
+        dc2 = this.state.nodes[1];
+    } else {
+        // eslint-disable-next-line prefer-destructuring
+        dc2 = this.state.nodes[0];
+    }
```

Comment:
This is unintuitive for what the step is doing, seems made to fit the scenarios we have here.
Could we make this a separate step, making it clearer and enabling us to use this verification in other scenarios?